frontend/encore/dev-server.rst

```diff
@@ -33,6 +33,40 @@ server SSL certificate:
     # Windows
     $ encore dev-server --https --pfx=%UserProfile%\.symfony\certs\default.p12
 
+To avoid repeating yourself, you can also update ``package.json`` to include
+these options automatically:
+
+.. code-block:: diff
+
+    {
+        ...
+        "scripts": {
+    -        "dev-server": "encore dev-server",
+    +        "dev-server": "encore dev-server --https --pfx=$HOME/.symfony/certs/default.p12 --allowed-hosts=mydomain.wip",
+            ...
+        }
+    }
+
+If you experience issues related to CORS (Cross Origin Resource Sharing), add
+the ``--disable-host-check`` and ``--port`` options to the ``dev-server``
```

Comment:
```suggestion
the ``--firewall false`` and ``--port`` options to the ``dev-server``
```

Reply:
`disable-host-check` has been removed in favor on `firewall` option, see https://github.com/webpack/webpack-dev-server/blob/master/CHANGELOG.md#firewall